#### 1.Write a code to draw a sample of values for X with size n where X have  a poisson distribution

In [1]:
from scipy.stats import poisson
import numpy as np

In [19]:
def pois(lamda,size):
    return np.random.poisson(lamda,size)

x=np.random.poisson(2,100)




#### 2.Compute the empirical mean

In [20]:
np.mean(x)

1.72

In this case the empirical is 1.72 who is less than 2 the mean of X

#### 4.Write a code to represent on the same graph the exact probability and its approximation.